### PR TITLE
Predicting Best Ball Playoff Teams with the Best Ball Value Curve and a Stacked XGBoost Model

### PR DESCRIPTION
**Pull request by philtousi**
Title: Predicting Best Ball Playoff Teams with the Best Ball Value Curve and a Stacked XGBoost Model

Participant: Philippe Tousignant

Contributions:

- The Best Ball Value Curve, a negative exponential curve explaining the relationship the between overall pick number and the pick points.
- A Stacked XGBoost Model predicting playoff teams with 97% ROC-AUC score and 95% accuracy.
- An analysis of the most predictive features of the model, and how it can assess individual roster.

Concluding remarks:

The outcomes of this exercise have yielded noteworthy results. The Best Ball Value Curve is theoretically sound and appears to be highly predictive. The model we have constructed exhibits high levels of discrimination and accuracy. It could be implemented, along with its result analysis methodology, on the Underdog website with some fine-tuning and few heuristic adjustments. Should this transpire, I would be delighted to offer my contribution to the project. In addition, the SHAP values provide intuitive insights into how the model predicts playoff teams and validate several fantasy football heuristics, including the significance of YAC and running backs with strong pass-catching abilities.

### CODE REVIEW

**Review by bendominguez0111 — COMMENTED** [review]
Can you add %%capture to the top of the cell where you import libraries and rerun?

See here if unfamiliar: https://notebook.community/lifeinoppo/littlefishlet-scode/RES/REF/python_sourcecode/ipython-master/examples/IPython%20Kernel/Capturing%20Output
